src/mockSdk.js

```diff
@@ -288,6 +300,7 @@ export default function createMockSDK(api = {}) {
     },
     people: {
       get: jest.fn(() => Promise.resolve(mockSDKPerson)),
+      list: jest.fn(() => Promise.resolve(mockSDKPersonList)),
```

Comment:
```suggestion
      list: jest.fn(() => Promise.resolve({items: [mockSDKPerson]})),
```

Reply:
I have added the people.list  mock function in the peopleSDKAdapter.test.js file as adding the people.list function in the mock sdk is resulting in the failure of test case in MeetingsSDKAdapter.test.js file.